velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -813,34 +813,6 @@ void persistReproInfo(
   }
 }
 
-std::unique_ptr<ReferenceQueryRunner> setupReferenceQueryRunner(
```

Comment:
nit: Can the include of DuckQueryRunner.h and PrestoQueryRunner.h in this file be removed too?

Reply:
> nit: Can the include of DuckQueryRunner.h and PrestoQueryRunner.h in this file be removed too?

Good point. Removed bunch of other includes as well.